src/browser/Navigators/Navigator/WebView/MozBrowserFrame.js

```diff
@@ -185,7 +185,7 @@ const decodeLocationChange =
     , canGoForward: null
     }
   : { type: "LocationChanged"
-    , uri: detail.uri
+    , url: detail.url
```

Comment:
@paulrouget Thanks! Will do that now. As for Servo's side of the URI to URL story, I am unable to run tests on my system. I am looking to contact team Servo for help with this.